### PR TITLE
FSP updates

### PR DESCRIPTION
**Pull request by out0xb2**
Update SplitFspBin.py for Python 3.x compat
Add path_env to enable edk2 Stuart build system

### CODE REVIEW

**Comment by out0xb2** [comment]
SplitFspBin.py updates for Python 3.x are incomplete... evaluating options...

**Comment by out0xb2** [comment]
FSP's successfully rebased after commit 36bc21fa8f419880ed9e2ca54c011f6d89b82017

File: IntelFsp2Pkg/Tools/SplitFspBin.py [hunk 1]
```diff
@@ -12,6 +12,7 @@
 import struct
 import argparse
 from   ctypes import *
+from functools import reduce
```
Comment (matthewfcarlson) [inline IntelFsp2Pkg/Tools/SplitFspBin.py:15]: Do you still need this? I don't see it being used

**Review by matthewfcarlson — APPROVED** [review]

File: IntelFsp2Pkg/Tools/SplitFspBin.py (hunk 1 above)
Comment (out0xb2) [reply IntelFsp2Pkg/Tools/SplitFspBin.py:15]: reduce is used by pre-existing, unmodified "Bytes2Val".  Python 3 requires this new import to call it.

File: IntelFsp2Pkg/Tools/fsp_tools_path_env.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+  "scope": "intel_fsp",
+  "flags": ["set_path", "set_pypath"]
```
Comment (corthon) [inline IntelFsp2Pkg/Tools/fsp_tools_path_env.json]: Do you need both path and pypath?

File: requirements.txt [hunk 3]
```diff
@@ -1,2 +1,2 @@
-edk2-pytool-library
-edk2-pytool-extensions
\ No newline at end of file
+edk2-pytool-extensions==0.9.5
```
Comment (corthon) [inline requirements.txt]: Conflict needs to be resolved with current requirements.txt.

**Review by corthon — CHANGES_REQUESTED** [review]

File: IntelFsp2Pkg/Tools/fsp_tools_path_env.json (hunk 2 above)
Comment (out0xb2) [reply IntelFsp2Pkg/Tools/fsp_tools_path_env.json]: Update in fifth Commit df8c67ce8a129fc03d9c174f3e246404fb05736f

**Comment by corthon** [comment]
Are you running with 201908? Can you resubmit this PR against that branch?

**Review by corthon — DISMISSED** [review]

**Comment by out0xb2** [comment]
I'll abandon and bring this to a newer branch

**Pull request closed without merging**